src/DataWrangling/ETOPO/ETOPO.jl

```diff
@@ -0,0 +1,84 @@
+module ETOPO
+
+using Scratch
+export ETOPOBathymetry
+
+import ClimaOcean.DataWrangling:
+    metadata_filename,
+    default_download_directory,
+    metadata_path,
+    Metadata,
+    Metadatum,
+    all_dates,
+    first_date,
+    last_date,
+    download_dataset,
+    download_progress,
+    longitude_interfaces,
+    latitude_interfaces,
+    is_three_dimensional,
+    z_interfaces,
+    reversed_vertical_axis
+
+using Downloads
+
+using Oceananigans
+using Oceananigans.DistributedComputations: @root
+
+download_ETOPO_cache::String = ""
+function __init__()
+    global download_ETOPO_cache = @get_scratch!("ETOPO")
+end
+
+ETOPO_bathymetry_variable_names = Dict(
+    :temperature => "temperature",
+    :salinity    => "salinity"
+)
+
+struct ETOPOBathymetry end
+
+default_download_directory(::ETOPOBathymetry) = download_ETOPO_cache
+reversed_vertical_axis(::ETOPOBathymetry) = true
+longitude_interfaces(::ETOPOBathymetry) = (-180, 180)
+latitude_interfaces(::ETOPOBathymetry) = (-90, 90)
+Base.size(dataset::ETOPOBathymetry, variable) = (21600, 10800, 1)
+Base.size(dataset::ETOPOBathymetry) = (21600, 10800, 1)
+
+all_dates(dataset::ETOPOBathymetry) = nothing
+all_dates(dataset::ETOPOBathymetry, variable) = nothing
+function first_date(dataset::ETOPOBathymetry, variable)
+    return nothing
+end
+last_date(dataset::ETOPOBathymetry, variable) = nothing
+
+const ETOPOMetadatum = Metadatum{<:ETOPOBathymetry, <:Any, <:Any}
```

Comment:
```suggestion
const ETOPOMetadatum = Metadatum{<:ETOPOBathymetry}
```
`Metadatum` has only one type parameter (see the definition in `metadata.jl`)